### PR TITLE
Add snake kebab

### PR DESCRIPTION
**Pull request by Nimfar11**
## About the PR

Adds snake kebabs and a recipe for cooking them.

## Why / Balance

Now the snake meat can be used for something.

## Media

изображение

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl: Nimfar11

- add: Adds snake kebab and its recipe.

### CODE REVIEW

**Comment by TheShuEd** [comment]
I want to inform you: in the future it is planned to rework the mechanics of cooking, and microwave recipes, including your work, may be deleted

**Comment by Nimfar11** [comment]
That future may come in a few months, at least now this meat will make sense.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/skewer.yml [hunk 1]
```diff
@@ -157,3 +157,25 @@
...
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 10
+        - ReagentId: Oculine
+          Quantity: 2
+        - ReagentId: Tricordrazine
+          Quantity: 2
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/skewer.yml:181]: why does it have a bunch of healing meds in it?

Comment (Nimfar11) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/skewer.yml:181]: Oculine is like the effect of the cobra having the blindness toxin, intended that way, but maybe there will be more snakes... Yeah, I'll remove the medicine. There's always the thought of giving some kind of feature in my head.

**Pull request merged**